tests/python/frontend/onnx/test_forward.py

```diff
@@ -5774,6 +5774,139 @@ def repeat(N, D):
     )
 
 
+@tvm.testing.parametrize_targets
```

Comment:
in unsupported_onnx_tests can you remove 

    "test_scan9_sum",
    "test_scan_sum",

And rerun `pytest ... test_onnx_nodes:test_scan_sum` `pytest ... test_onnx_nodes:test_scan9_sum`

Reply:
I have remove them in unsupported_onnx_tests, but could not run test_onnx_nodes[test_scan_sum] and test_onnx_nodes[test_scan9_sum] in docker ci_gpu, it said not found.

Reply:
Interesting. Well the tests fail so maybe you can try running locally with the latest version of onnx and onnxruntime. 

We should pass the official tests to guarantee correctness with the onnx spec.